plugins/auto-sizes/tests/test-improve-sizes.php

```diff
@@ -26,6 +26,21 @@ public static function set_up_before_class(): void {
 		self::$image_id = self::factory()->attachment->create_upload_object( TESTS_PLUGIN_DIR . '/tests/data/images/leaves.jpg' );
 	}
 
+	public function set_up(): void {
+		parent::set_up();
+
+		// Disable auto sizes.
+		remove_filter( 'wp_content_img_tag', 'auto_sizes_update_content_img_tag' );
+	}
+
+	public function tear_down(): void {
+		// Clean up specific to the tests within this class.
+		parent::tear_down();
+
+		// Enable auto sizes.
+		add_filter( 'wp_content_img_tag', 'auto_sizes_update_content_img_tag' );
+	}
+
```

Comment:
This shouldn't be necessary because WordPress restores all default filters and actions with each test already.

```suggestion
```